codebase/general/src.bin/task/eval.1.4/eval.c

```diff
@@ -247,6 +247,10 @@ int main(int argc,char *argv[]) {
   else decode_format(format);
   store[0]=PI;
   s=Eval(argv[arg],&val,decode_value,store,decode_function,store);
+  if (s == -1)
+  {
+      fprintf(stderr, "Warning: Eval function returned a negative may have an error\n");
```

Comment:
So Eval doesn't have a return -1 but a calculated number but then we don't use it ... this is where a function description would be handy to know what it is doing. So I didn't want to fail the code if it might send -1. Does that make sense?